ui/src/app/machines/hooks.tsx

```diff
@@ -1,7 +1,7 @@
 import { useCallback } from "react";
 import { useSelector } from "react-redux";
 
-import { ACTIONS } from "app/base/reducers/machine/machine";
+import { ACTIONS } from "app/store/machine/slice";
```

Comment:
Is there a reason this is uppercase? It isn't a global const, so looks a bit odd.